.github/workflows/codeql-analysis.yml

```diff
@@ -2,9 +2,8 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ main, v* ]
+    branches-ignore: dependabot/**
```

Comment:
Admittedly not sure if this would resolve the errors as it appears to be failing on `push` events to `main` - https://github.com/ScottBrenner/cfn-lint-action/actions/workflows/codeql-analysis.yml?query=event%3Apush

However https://github.com/ScottBrenner/cfn-lint-action/runs/2764966785?check_suite_focus=true#step:3:55 _does_ say `avoid triggering on the "push" event for Dependabot branches`